src/Stratis.Bitcoin/EventBus/GenericEventBase.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Stratis.Bitcoin.EventBus
+{
+    /// <summary>
+    /// Basic implementation of a generic <see cref="EventBase"/>.
+    /// </summary>
+    /// <seealso cref="Stratis.Bitcoin.EventBus.EventBase" />
+    public class GenericEventBase<TContent> : EventBase
```

Comment:
ah there is one like this I would rename to just EventBase<TContent>

Reply:
See comment above.
This class is more a simple way to define an event with a Content typed property, it's not meant to be used as it is (btw I should mark this as abstract too)